src/bp-core/bp-core-update.php

```diff
@@ -369,6 +369,14 @@ function bp_version_updater() {
 		if ( $raw_db_version < 18701 ) {
 			bb_update_to_1_9_3();
 		}
+
+		if ( $raw_db_version < 18751 ) {
+			bb_update_to_2_1_0();
+		}
+
+		if ( $raw_db_version < 18801 ) {
+			bb_update_to_2_2_0();
+		}
```

Comment:
@milanhirapra we need to make this to a single function as now all the things are on 1948 branch.